README.md

```diff
@@ -47,71 +47,23 @@ You can use the create react app setup in `playground/nextjs` to test posthog-js
 2. Cypress tests - integrates with a real chrome browser and is capable of testing timing, browser requests, etc. Useful for testing high-level library behavior, ordering and verifying requests. We shouldn't aim for 100% coverage here as it's impossible to test all possible combinations.
 3. TestCafe E2E tests - integrates with a real posthog instance sends data to it. Hardest to write and maintain - keep these very high level
 
-## Developing together with another repo
-
-#### Developing with main PostHog repo
-
-The `posthog-js` snippet for a website loads static js from the main `PostHog/posthog` repo. Which means, when testing the snippet with a website, there's a bit of extra setup required:
-
-1. Run `PostHog/posthog` locally
-2. Link the `posthog-js` dependency to your local version (see below)
-3. Run `yarn start` in `posthog-js`. (This ensures `dist/array.js` is being generated)
-4. In your locally running `PostHog/posthog` build, run `yarn copy-scripts`. (This copies the scripts generated in step 3 to the static assets folder for `PostHog/posthog`)
-
-Further, it's a good idea to modify `start-http` script to add development mode: `webpack serve --mode development`, which doesn't minify the resulting js (which you can then read in your browser).
-
-### Using Yalc to link local packages
```

Comment:
I'd be against removing this unless we have an alternative. For now, this is pretty wonky, but works 😬